src/mococrw/padding_mode.h

```diff
@@ -224,7 +225,7 @@ class PSSPadding: public RSAPadding {
      * @param Unused
      * @return -1.
      */
-    int getDataMaxSize(const AsymmetricPublicKey& key) const override
+    int getDataMaxSize(AsymmetricPublicKey& key) const override
     {
         /*Ignore unused parameter*/
         std::ignore = key;
```

Comment:
side remark. In modern C++, leaving the name of the parameter empty suffices to indicate that you know that the parameter will be unused.